### PR TITLE
Update docs on the output of `resolve qlpacks`

### PR DESCRIPTION
**Pull request by aeisenberg**
The output has changed and there are no more upgrades
packs. There are also other changes included here.

Related to https://github.com/github/semmle-code/pull/40918

### CODE REVIEW

File: docs/codeql/codeql-cli/getting-started-with-the-codeql-cli.rst [hunk 1]
```diff
@@ -212,16 +212,20 @@ up to create and analyze databases:
...
+
+  - Query packs for each supported language, e.g., ``codeql/{language}-queries``.
+    These packs contain the standard queries that will be run for each analysis.
+  - Library packs for each supported language, e.g.,  ``codeql/{language}-all``. These
+    packs contain query libraries, such as control flow and data flow libraries, that
+    may be useful to other query authors.
+  - Example packs for each supported language, e.g., ``codeql/{language}-examples``.
+    These packs contain useful snippets of CodeQL that query auhtors may want To use.
+  - Legacy packs that ensure custom queries and libraries created using older products are
+    compatible with your version of CodeQL.
```
Comment (felicitymay) [inline docs/codeql/codeql-cli/getting-started-with-the-codeql-cli.rst:228]: Some minor editorial fixes:

```suggestion
  - Query packs for each supported language, for example, ``codeql/{language}-queries``.
    These packs contain the standard queries that will be run for each analysis.
  - Library packs for each supported language, for example,  ``codeql/{language}-all``. These
    packs contain query libraries, such as control flow and data flow libraries, that
    may be useful to query writers.
  - Example packs for each supported language, for example, ``codeql/{language}-examples``.
    These packs contain useful snippets of CodeQL that query writers may find useful.
  - Legacy packs that ensure custom queries and libraries created using older products are
    compatible with your version of CodeQL.
```

**Review by felicitymay — DISMISSED** [review]
Many thanks for the docs update ✨ 

I've made a suggestion with some minor editorial fixes, but otherwise this looks ready to merge.

Presumably this will ship with the CodeQL CLI release that's expected at the end of January/early February, but can be merged now?

**Comment by aeisenberg** [comment]
> Presumably this will ship with the CodeQL CLI release that's expected at the end of January/early February, but can be merged now?

Yes. The change doesn't mention the upgrades packs. These packs exist in CLI 2.7.5 and will be removed in 2.7.6.

**Review by felicitymay — APPROVED** [review]

**Comment by aeisenberg** [comment]
v2.7.6 is released now. I will merge these docs changes so they will be released along with v2.8.0

**Pull request merged**